lib/player.rb

```diff
@@ -0,0 +1,33 @@
+class Player
+  attr_reader :name, :symbol
+  attr_accessor :num_of_wins
```

Comment:
[__OPTIONAL__]
Make a function that will set this attribute so there will be no needs for `attr_accessor` at all